src/pages/market/components/ItemCard.jsx

```diff
@@ -0,0 +1,19 @@
+import { ReactComponent as HeartIcon } from '../../../images/icons/ic_heart.svg';
+
+function ItemCard({ item }) {
+  return (
+    <div className='itemCard'>
+      <img src={item.images[0]} alt={item.name} className='itemCardThumbnail' />
+      <div className='itemSummary'>
+        <h2 className='itemName'>{item.name}</h2>
+        <p className='itemPrice'>{item.price.toLocaleString()}원</p>
```

Comment:
```suggestion
        <p className='itemPrice'>{item.price.toLocaleString("ko-KR")}원</p>
```
꿀팁 공유~
number.toLocaleString에 첫번째 인자로 "ko-KR"을 넘겨주면 천 단위마다 자동으로 콤마를 찍어주는 기능이 있어요!:-)